webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7355,44 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.getBotDetails = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/bot/" + MCK_APP_ID,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                    var flag = false;
+                    console.log(data);
+                    for (var key in data)
+                    {
+                        if (data.hasOwnProperty(key)) {
+                            if(data[key].name == CURRENT_GROUP_DATA.conversationAssignee){
+                                //console.log(data[key].name);
+                                if(data[key].aiPlatform == 'dialogflow')
+                                {
+                                    if(!data[key].autoHumanHandoff){
+                                        CHAR_CHECK = true;
+                                        document.getElementsByClassName('mck-char-warning')[0].style.backgroundColor = WIDGET_SETTINGS.primaryColor;
+                                        break;
+                                    }else{
+                                        CHAR_CHECK = false;
```

Comment:
@hardikparashar multiple else not required if all have it as 'false' only. keep the default value as false before this block of code executes and remove all else part where 'false' is set.

Reply:
done

Reply:
resolved in new commit